warehouse/templates/manage/token.html

```diff
@@ -0,0 +1,95 @@
+{#
+ # Licensed under the Apache License, Version 2.0 (the "License");
+ # you may not use this file except in compliance with the License.
+ # You may obtain a copy of the License at
+ #
+ # http://www.apache.org/licenses/LICENSE-2.0
+ #
+ # Unless required by applicable law or agreed to in writing, software
+ # distributed under the License is distributed on an "AS IS" BASIS,
+ # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ # See the License for the specific language governing permissions and
+ # limitations under the License.
+-#}
+{% extends "manage_base.html" %}
+
+{% set user = request.user %}
+{% set title = "Add API token" %}
+
+{% block account_active %}vertical-tabs__tab--is-active{% endblock %}
+{% block account_mobile_active %}vertical-tabs__tab--is-active{% endblock %}
+
+{% block title %}
+  {% if create_macaroon_form.errors %}Error processing form –{% endif %}
+  {{ title }}
+{% endblock %}
+
+{% block main %}
+  <h1 class="page-title">{{ title }} <span class="badge badge--warning">Beta</span></h1>
+
+  {% if serialized_macaroon %}
+  <section id="provisioned-key">
+    <h2>Token for "{{ macaroon.description }}"</h2>
+    <p>For instructions on how to use this token, <a href="/help#apitoken">visit the PyPI help page</a>.</p>
+    <code class="api-token">{{ serialized_macaroon }}</code>
+    <div class="margin-bottom--large">
+      <div class="callout-block callout-block--danger no-top-margin">
+        <p>For security reasons this token will only appear once. <strong>Copy it now.</strong></p>
+      </div>
+      <button class="button button--primary -js-copy-hash tooltipped tooltipped-e" aria-label="Copy token to clipboard" data-original-label="Copy token to clipboard" data-clipboard-text="{{ serialized_macaroon }}">
+        Copy
+      </button>
+      <a href="#remove-API-token--{{ macaroon.id }}" class="button">
+        Remove
+      </a>
+      {% set slug="remove-API-token--" + macaroon.id | string %}
+      {% set title="Remove API token - " + macaroon.description %}
+      {% set confirm_button_label="Remove API token" %}
+      {% set extra_fields %}
+        <input type="text" name="macaroon_id" value="{{ macaroon.id }}" hidden>
+      {% endset %}
+      {% set token_warning_text %}
+        <p>Applications or scripts using this token will no longer have access to PyPI.</p>
+      {% endset %}
+
+      {{ confirm_modal(
+        title=title,
+        confirm_name="Username",
+        confirm_string=user.username,
+        confirm_button_label=confirm_button_label,
+        slug=slug,
+        extra_fields=extra_fields,
+        custom_warning_text=token_warning_text,
+        confirm_string_in_title=False) }}
```

Comment:
@woodruffw This is what's causing the `lint` failure, this should all be a single line (FYI, `{{` and `}}` are called 'echo' tags)

Reply:
Thanks!